blueprints/controllers/ikea_e1812/ikea_e1812.yaml

```
@@ -1,40 +1,51 @@
 # Blueprint metadata
 blueprint:
   name: Controller - IKEA E1812 Shortcut button
-  description: |
-    Controller automation for executing any kind of action triggered by an IKEA E1812 Shortcut button. Allows to optionally loop an action on a button long press.
+  description: 'Controller automation for executing any kind of action triggered by
+    an IKEA E1812 Shortcut button. Allows to optionally loop an action on a button
+    long press.
+
     Supports deCONZ, ZHA, Zigbee2MQTT.
 
-    Automations created with this blueprint can be connected with one or more [Hooks](https://epmatt.github.io/awesome-ha-blueprints/docs/blueprints/hooks) supported by this controller.
-    Hooks allow to easily create controller-based automations for interacting with media players, lights, covers and more.
+
+    Automations created with this blueprint can be connected with one or more [Hooks](https://epmatt.github.io/awesome-ha-blueprints/docs/blueprints/hooks)
+    supported by this controller.
+
+    Hooks allow to easily create controller-based automations for interacting with
+    media players, lights, covers and more.
+
 
     Full documentation for this controller and list of available Hooks [here](https://epmatt.github.io/awesome-ha-blueprints/docs/blueprints/controllers/ikea_e1812).
-  source_url: https://github.com/EPMatt/awesome-ha-blueprints/blob/main/blueprints/controllers/ikea_e1812/ikea_e1812.yaml
+
+    '
+  source_url: https://github.com/EPMatt/awesome-ha-blueprints/blob/issue-22/blueprints/controllers/ikea_e1812/ikea_e1812.yaml
   domain: automation
   input:
     integration:
       name: Integration
-      description: Integration used for connecting the remote with Home Assistant. Select one of the available values.
+      description: Integration used for connecting the remote with Home Assistant.
+        Select one of the available values.
       selector:
         select:
           options:
-            - deCONZ
-            - ZHA
-            - Zigbee2MQTT
+          - deCONZ
+          - ZHA
+          - Zigbee2MQTT
```

```suggestion
            - deCONZ
            - ZHA
            - Zigbee2MQTT
```

Keeping the current format here